docs/content/guides/rows/row-sorting.md

```diff
@@ -453,163 +1184,2251 @@ import 'handsontable/dist/handsontable.full.min.css';
+            model: 'Road Tire Tube',
+            price: 59,
+            sellDate: '28/08/2023',
+            sellTime: '08:01',
+            inStock: true,
+          },
+          {
+            brand: 'Chatterpoint',
+            model: 'HL Road Tire',
+            price: 279.99,
+            sellDate: '02/10/2023',
+            sellTime: '13:23',
+            inStock: true,
+          },
+        ]}
+        columns={[
+          {
+            title: 'Brand',
+            type: 'text',
+            data: 'brand',
+          },
+          {
+            title: 'Model',
+            type: 'text',
+            data: 'model',
+          },
+          {
+            title: 'Price',
+            type: 'numeric',
+            data: 'price',
+            numericFormat: {
+              pattern: '$ 0,0.00',
+              culture: 'en-US'
+            },
+            className: 'htLeft',
+          },
+          {
+            title: 'Date',
+            type: 'date',
+            data: 'sellDate',
+            className: 'htRight',
+          },
+          {
+            title: 'Time',
+            type: 'time',
+            data: 'sellTime',
+            className: 'htRight',
+          },
+          {
+            title: 'In stock',
+            type: 'checkbox',
+            data: 'inStock',
+            className: 'htCenter',
+          },
+        ]}
+        multiColumnSorting={true}
+        height="auto"
+        width="auto"
+        licenseKey="non-commercial-and-evaluation"
+      />
+      <div className="controls">
+        <button onClick={sort}>Sort</button>
+      </div>
+    </>
+  );
+}
+
+/* start:skip-in-preview */
+ReactDOM.render(<HandsontableComponent />, document.getElementById('example8'));
+/* end:skip-in-preview */
+```
+
+:::
+
+:::
+
+### Exclude rows from sorting
+
+You can exclude any number of top or bottom rows from sorting.
+
+To do this, use Handsontable's [`afterColumnSort`](@/api/hooks.md#aftercolumnsort) hook,
+which is fired after each sorting. Inside the hook, call the [`moveIndexes()`](@/api/indexMapper.md#moveindexes) method
+to change the visual indexes of the rows that you want to exclude.
+
+::: only-for javascript
+
+```js
+const configurationOptions = {
+  afterColumnSort() {
+    // keep rows number 1 and 2 at index 0
+    handsontableInstance.rowIndexMapper.moveIndexes(
+      [
+        handsontableInstance.toVisualRow(0),
+        handsontableInstance.toVisualRow(1),
+      ], 0);
+
+```
+
+:::
+
+::: only-for react
+```jsx
+const exclude = () => {
+  // keep rows number 1 and 2 at index 0
+  hotTableComponentRef.current.hotInstance.rowIndexMapper.moveIndexes(
+    [
+      hotTableComponentRef.current.hotInstance.toVisualRow(0),
+      hotTableComponentRef.current.hotInstance.toVisualRow(1),
+    ], 0);
+};
+
+<HotTable
+  afterColumnSort={exclude}
+/>
+```
+:::
+
+In this demo, click on any column name. The rows get sorted, but the the first row and the last row stay in place.
```

Comment:
Please rewrite this sentence to avoid generic sentences like "In this demo, click on any column name.".